recipes/npcap/conanfile.py

```diff
@@ -0,0 +1,43 @@
+from os.path import join
+from conan import ConanFile
+from conan.tools.files import get, download, copy
+
+
+class NpcapConan(ConanFile):
+    name = "npcap"
+    description = "Windows port of the libpcap library"
+    homepage = "https://npcap.com/"
+    url = "https://github.com/conan-io/conan-center-index"
+    license = "https://github.com/nmap/npcap/blob/master/LICENSE"
+    topics = ("npcap", "pcap")
+    settings = {
+        "os": ["Windows"],
+        "arch": ["x86", "x86_64", "armv8"],
+        "build_type": None,
+    }
```

Comment:
```suggestion
    settings = "os", "arch", "build_type", "compiler"
```
Keep full settings always. You can manage the package ID